stores/sql.go

```diff
@@ -230,6 +205,44 @@ func isSQLite(db *gorm.DB) bool {
 	}
 }
 
+func performMigrations(db *gorm.DB) error {
+	// Perform manual migrations.
+	if err := db.Migrator().DropTable("host_sectors"); err != nil {
```

Comment:
This is pretty great. I checked and it supports index handling as well:

```
// Indexes
  CreateIndex(dst interface{}, name string) error
  DropIndex(dst interface{}, name string) error
  HasIndex(dst interface{}, name string) bool
  RenameIndex(dst interface{}, oldName, newName string) error
  ```
  
  so we probably want to use this opportunity to get rid of some of these in `sqlstore.go`:
  ```go
  // Ensure the join table has an index on `db_host_id`.
	switch conn.(type) {
	case *sqlite.Dialector:
		if err := db.Exec("CREATE INDEX IF NOT EXISTS idx_host_blocklist_entry_hosts ON host_blocklist_entry_hosts (db_host_id)").Error; err != nil {
			return nil, modules.ConsensusChangeID{}, err
		}
	case *mysql.Dialector:
		var found int
		err := db.Raw("SELECT COUNT(1) IndexIsThere FROM INFORMATION_SCHEMA.STATISTICS WHERE table_schema=DATABASE() AND table_name='host_blocklist_entry_hosts' AND index_name='idx_host_blocklist_entry_hosts'").Scan(&found).Error
		if err != nil {
			return nil, modules.ConsensusChangeID{}, err
		}
		if found == 0 {
			if err := db.Exec("CREATE INDEX idx_host_blocklist_entry_hosts ON host_blocklist_entry_hosts (db_host_id)").Error; err != nil {
				return nil, modules.ConsensusChangeID{}, err
			}
		}
	default:
		panic("unknown dialector")
	}
  ```